### PR TITLE
persist settings in query UI

### LINKED ISSUE #1743: query: UI doesn't persist Thanos-specific settings
When using the Thanos query UI, the additional settings like "deduplication", "partial response", etc are not persisted to the URL the same way settings like "stacked" are. This makes it difficult to share URLs.

Also problematic is that the UI defaults to "Only raw data", which can be expensive on large queries.

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

* [ ] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

## Changes

Fixes #1743

### CODE REVIEW

File: pkg/ui/templates/query_menu.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
           <ul class="navbar-nav">
             <li class="nav-item"><a class="nav-link" href="{{ pathPrefix }}/graph">Graph</a></li>
             <li class="nav-item"><a class="nav-link" href="{{ pathPrefix }}/stores">Stores</a></li>
-	    <li class="nav-item dropdown">
+	        <li class="nav-item dropdown">
```
Comment (squat) [inline pkg/ui/templates/query_menu.html]: I think this indentation is unneeded

Comment (yeya24) [reply pkg/ui/templates/query_menu.html]: Thanks! I just format this page.

**Comment by yeya24** [comment]
Sorry, just forget to make the two options as default, let me modify this PR.

File: pkg/ui/templates/query_menu.html (hunk 1 above)
Comment (squat) [reply pkg/ui/templates/query_menu.html]: thanks (: looks like we have a bit of a mix of 2sp and 4sp in those templates. I'll make a followup to standardize it.

Comment (GiedriusS) [reply pkg/ui/templates/query_menu.html]: :+1:

**Pull request merged**